spec/unit/libraries/path_helpers_spec.rb

```diff
@@ -1,9 +1,11 @@
-require_relative 'spec_helper'
+require_relative '../spec_helper'
```

Comment:
Ah right, I had it working when I moved all the specs into the root directory before I found out about the whole foodcritic thing and we put them all under /unit.  disregard my comment.